kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveDriver.java

```diff
@@ -33,48 +33,31 @@
 import java.net.InetSocketAddress;
 import java.util.Map;
 import java.util.Properties;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.ScheduledThreadPoolExecutor;
-import org.apache.kafka.clients.admin.Admin;
-import org.apache.kafka.common.config.TopicConfig;
 import org.apache.kafka.common.config.types.Password;
-import org.apache.kafka.common.utils.Bytes;
-import org.apache.kafka.streams.kstream.Materialized;
-import org.apache.kafka.streams.state.KeyValueBytesStoreSupplier;
-import org.apache.kafka.streams.state.KeyValueStore;
-import org.apache.kafka.streams.state.WindowBytesStoreSupplier;
-import org.apache.kafka.streams.state.WindowStore;
+import org.apache.kafka.streams.StreamsConfig;
+
 
 /**
  * The {@code ResponsiveDriver} should be instantiated once per JVM
  * and maintains a session and connection to the remote storage server.
  */
-// TODO(agavra): we should put more thought into this API and consider splitting
-// it up into a "reusable" session class and a "per-streams" driver so that we
-// can properly track resources created by the driver
-public class ResponsiveDriver implements StreamsStoreDriver, Closeable {
-
-  private static final Map<String, String> CHANGELOG_CONFIG = Map.of(
-      TopicConfig.CLEANUP_POLICY_CONFIG, TopicConfig.CLEANUP_POLICY_DELETE);
+public class ResponsiveDriver implements Closeable {
```

Comment:
Maybe should be renamed to `ResponsiveSession` or something like that, but didn't want to add unnecessary review burden. Can save any name changes for just before merging